assets/sass/vendor/_mdc-dialog.scss

```diff
@@ -2,22 +2,69 @@
 .googlesitekit-plugin .mdc-dialog {
 	z-index: 10000;
 
+	%label-small {
+		font-size: $fs-body-sm;
+		letter-spacing: 0.2px;
+		line-height: $lh-label-sm;
+	}
+
 	.mdc-dialog__surface {
-		padding: $grid-gap-desktop $grid-gap-desktop * 2;
+		border-radius: 8px;
+		box-shadow: 0 10px 40px 0 rgba(#000, 0.35);
+		padding: $grid-gap-desktop + 12px $grid-gap-desktop * 2 - 10; // 36px 38px.
```

Comment:
Hm, I was sure I saw 38px in one of the modals I was taking values from. Now I checked 2 of them, and it is 36px everywhere. I was mistaken for some reason. Will update it to 36px and use the real value as you suggested, it will be more readable. Thanks